gitguard.py

```diff
@@ -0,0 +1,51 @@
+import re
+import subprocess
+import github
+
+"""
+gitguard_extractor.py
+
+Extracts data for the visualizer.
+
+repo_link is in the format USER/REPO_NAME or ORGANIZATION/REPO_NAME
+"""
+
+REGEX_REPO_LINK_DELIMITER = '\s*/\s*'
+
+def process_repo_link(repo_link):
+    #returns owner, repo_name
+    return re.compile(REGEX_REPO_LINK_DELIMITER).split(repo_link)
+
+def _get_contributors_from_api(repo_link):
+    owner, repo = process_repo_link(repo_link)
+    # connect to github API
+    gh = github.GitHub()
+    return gh.repos(owner)(repo).contributors.get() 
```

Comment:
maybe you want to refactor into "get_repo(repo_link)" since the line above this are repeated.